pylearn2/models/mlp.py

```diff
@@ -2961,14 +2967,14 @@ def __init__(self,
 
         if pool_type is not None:
             assert pool_shape is not None, (
-                "You should specify the shape of "
-                "the spatial %s-pooling." % pool_type)
+                "You should specify the shape of the spatial %s-pooling." %
+                pool_type)
             assert pool_stride is not None, (
-                "You should specify the strides of "
-                "the spatial %s-pooling." % pool_type)
+                "You should specify the strides of the spatial %s-pooling." %
+                pool_type)
 
         assert nonlinearity is not None
-
+        super(ConvElemwise, self).__init__()
```

Comment:
If I am not wrong, it was like that in the code I was porting.
It seems reasonable to check the consistency of the parameters before calling the inherited constructor though, isn't it?